demosplan/DemosPlanCoreBundle/ResourceTypes/ProcedureMapSettingResourceType.php

```diff
@@ -255,9 +280,32 @@ protected function convertJsonToCoordinates(string $rawCoordinateValues): ?array
     /**
      * @return list<int>
      */
-    protected function getAvailableScales(): array
+    protected function getScales($scaleSettings): array
+    {
+        return $this->convertToListOfInt(str_replace(['[', ']'], '', (string) $scaleSettings));
+    }
+
+    /**
+     * @return list<int>
```

Comment:
Return type does not see correct. You could include AvailableProjectionVO here

Reply:
adjusted here 32ad47c4f3571ab8917f1e89ff32ef419e1ea3f8